data/mods/Aftershock/mutations/dreams.json

```diff
@@ -0,0 +1,35 @@
+[
+  {
+    "type": "dream",
+    "messages": [ "You dream of warm gusts of strange winds.  You look up, four moons gaze down on you." ],
```

Comment:
This could be better sereved by being two sepparate dreams. Something like
```
"You dream of warm, alien winds."
"You dream of landscapes lit by the light of alien moons."
```